projects/personachat/persona_seq2seq.py

```diff
@@ -1664,6 +1668,7 @@ def _encode(self, xs, is_training=False):
             if type(self.decoder) != nn.LSTM:
                 hidden = hidden[0]
         else:
+            xes_packed = pack_padded_sequence(xes.transpose(0, 1), x_lens)
```

Comment:
if you use pack_padded_sequence then I think you'll need to use pad_packed_sequence on the encoder_output

Reply:
does this actually run as is?